app.R

```diff
@@ -434,9 +440,9 @@ server <- function(input, output, session) {
         )
       })
       
-      if (!is.null(values$dataset)) {# We had previously loaded a file and need to display the new file
-        setColNames(2, 1) 
-      }
+      #if (!is.null(values$dataset)) {# We had previously loaded a file and need to display the new file
```

Comment:
If you're sure you won't want to keep this code (and have a backup somewhere), delete commented code to keep things clean.